src/pages/rest/modules/import/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: ImportCsv API
+
+description: Import entities into Adobe Commerce using Rest API
+
+--- 
+## Import API
+
+The  `StartImportInteface` service provides the means to efficiently import entities in Magento using a single API call. 
+
+The `POST /rest/default/V1/import/csv` call allows import of the following entities:
+
+* Advanced Pricing
+* Products (Simple & Configurable)
+* Customers
+* Customer Finances
+* Customer Addresses
+
+
+**Service Name:**
+
+`StartImportInterface`
+
+**REST Endpoint:**
+
+```http
+POST /rest/default/V1/import/csv
+```
+
+**StartImportInterface Parameters:**
+
+
+|Name | Description  | Format | Requirements|
+|-----|--------------|--------|---------------------------------------------------------------------------------|
+|`entity`|Type of entity to be imported| string | Required for determining type of entity being imported|
+|`behavior` | Import Behavior | string | Required for determining if entities are being added/updated,replaced or deleted |
+|`validationStrategy` | Strategy to use when entities are invalid | string | Required for determining validation strategy|
+|`allowedErrorCount` | No. of errors before import is canceled | string | Required for canceling import after certain number of invalid rows|
```

Comment:
```suggestion
|`allowedErrorCount` | The maximum number of errors that can occur until import is canceled | string | Required |
```

What happens with the uploads that did not encounter errors before the error count is reached? Are they marked as successful, or are they rolled back?